pkgs/development/python-modules/pysvg-py3/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "pysvg-py3";
+  version = "0.2.2.post3";
+
+  src = fetchFromGitHub {
+    owner = "alorence";
+    repo = pname;
+    rev = version;
+    sha256 = "0rm44i9lm7gpblwid8mhpr27g0cb20dycgbrw6g2fjq6qxrin7hj";
+  };
+
+  checkPhase = ''
+    runHook preCheck
+    mkdir testoutput
+    ${python.interpreter} sample/tutorial.py
+    runHook postCheck
+  '';
```

Comment:
```suggestion
  '';
  
  pythonImportsCheck = [ "pysvg" ];
```

Reply:
their example i run imports everything... do you think it still makes sense to test the import of the module as a whole?

https://github.com/alorence/pysvg-py3/blob/master/sample/tutorial.py#L5-L14

Reply:
i just added it...